### PR TITLE
Don't generate `config/spring.rb` in `app:update` task when spring isn't loaded

### PR DESCRIPTION
**Pull request by ttanimichi**
Related to #32780, #29645 and #28825

### CODE REVIEW

**Review by sikachu — APPROVED** [review]

File: railties/test/generators/app_generator_test.rb [hunk 1]
```diff
@@ -309,6 +309,17 @@ def test_app_update_does_not_generate_assets_initializer_when_skip_sprockets_is_
     end
   end
 
+  def test_app_update_does_not_generate_spring_contents_when_skip_spring_is_given
+    app_root = File.join(destination_root, "myapp")
+    run_generator [app_root, "--skip-spring"]
+
+    FileUtils.cd(app_root) do
+      assert(quietly { system("bin/rails app:update") })
```
[the remark this answers is not in the document]
Comment (ttanimichi) [reply railties/test/generators/app_generator_test.rb]: Okey, I give up to check the exit status.

[the remark this answers is not in the document]
Comment (ttanimichi) [reply railties/test/generators/app_generator_test.rb]: Fixed https://github.com/rails/rails/pull/32837/commits/ac336924cff389530f1f05893000eec59004b643

**Comment by y-yagi** [comment]
LGTM. Please squash your commits into a single commit?

**Comment by ttanimichi** [comment]
Squashed https://github.com/rails/rails/pull/32837/commits/8efa112ca625cf8e895ed5af1433dce430929970

**Comment by y-yagi** [comment]
Backported to [5-2-stable](3ea9832be872c2331cdcd3e0da120c52c3810b85).

**Pull request merged**